tests/ClientFunctionalTest.php

```diff
@@ -223,6 +227,28 @@ public function testSandboxesOnBranch(): void
         );
     }
 
+    public function testCreateSandboxWithSize(): void
+    {
+        $sandbox = (new Sandbox())
+            ->setActive(true)
+            ->setType('python')
+            ->setSize(Sandbox::CONTAINER_SIZE_SMALL)
+            ->setSizeParameters(new SandboxSizeParameters(128))
```

Comment:
I prefer the `fromArray` construction because that way you can see the fields and their values.

Reply:
I've change it to static `create` + setter call